packages/smooth_app/lib/helpers/temp_product_list_share_helper.dart

```diff
@@ -9,9 +9,10 @@ Uri shareProductList(
   final String barcodesString = barcodes.join(',');
 
   return UriHelper.replaceSubdomain(
-    ProductQuery.getUriProductHelper(
-      productType: productType,
-    ).getUri(path: 'products/$barcodesString', addUserAgentParameters: false),
+    ProductQuery.getUriProductHelper(productType: productType).getUri(
+      path: 'products/${Uri.encodeComponent(barcodesString)}',
```

Comment:
Not sure you shouldn't encode barcodes and then join them.
Depends on how a comma is considered in "advanced" barcodes.
Is there a unit test for that?